site/docs/spec.md

```diff
@@ -498,6 +498,7 @@ Table metadata consists of the following fields:
 | _required_ |            | **`partition-spec`**| The table’s current partition spec, stored as only fields. Note that this is used by writers to partition data, but is not used when reading because reads use the specs stored in manifest files. (**Deprecated**: use `partition-specs` and `default-spec-id`instead ) |
 | _optional_ | _required_ | **`partition-specs`**| A list of partition specs, stored as full partition spec objects. |
 | _optional_ | _required_ | **`default-spec-id`**| ID of the “current” spec that writers should use by default. |
+| _optional_ | _required_ | **`last-partition-id`**| An integer; the highest assigned partition field ID across all partition specs for the table. This is used to ensure partition fields are always assigned an unused ID when evolving specs. |
```

Comment:
Nit: might not need to mention integer since the JSON specifications section will include it, and also to maintain consistency with other rows mentioned in the table. Also "ensure partition fields are always assigned" -> "ensure new partition fields are always assigned to an"?

Reply:
This matches the description for last-column-id, so I think it's okay.